docs/admin/multiple-zones.md

```diff
@@ -0,0 +1,280 @@
+---
+---
+
+## Introduction
+
+Kubernetes 1.2 adds support for running a single cluster in multiple zones.
```

Comment:
Possibly add something like "simply referred to as "zones" in the remainder of this document".